### PR TITLE
.github: Consolidate triage workflows

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Fixes #{issue number}

### CODE REVIEW

File: .github/workflows/issue_triage.yml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+name: Triage issues to Projects
 
 on:
   issues:
     types: [labeled]
 
 jobs:
-  welcome:
+  triage-issues:
+    strategy:
```
Comment (samestep) [inline .github/workflows/issue_triage.yml:9]: maybe add this just in case?
```suggestion
    strategy:
      fail-fast: false
```

**Review by samestep — APPROVED** [review]

**Comment by samestep** [comment]
Test plan?

**Comment by seemethere** [comment]
> Test plan?

There's no really easy way to do this, so maybe we land, check if it works and then revert it doesn't?

**Comment by samestep** [comment]
> There's no really easy way to do this, so maybe we land, check if it works and then revert it doesn't?

Hmm yeah you're right; @janeyx99 and I have usually been testing PRs like this on sandbox repos before submitting them for review here on the PyTorch repo

**Pull request closed without merging**